app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which means it is shared across all instances of `Animal`. This is correct for tracking all alive animals, but ensure that this is the intended behavior.